data/ui_dashboards/molgenis.csv

```diff
@@ -0,0 +1,69 @@
+tableName,tableExtends,tableType,columnName,columnType,key,required,refTable,refBack,defaultValue,validation,semantics,description
+Dashboard pages,,,,,,,,,,,http://semanticscience.org/resource/SIO_000302,The user interfaces in the vue application and the charts are displayed in each view
+Dashboard pages,,,name,string,1,TRUE,,,,,http://schema.org/name,Name of the page
+Dashboard pages,,,description,text,,,,,,,http://schema.org/description,Text that describes the aim of the page
+Dashboard pages,,,charts,refback,,,Charts,chart id,,,http://edamontology.org/topic_0092,One or more charts displayed on a page
+Charts,,,,,,,,,,,http://semanticscience.org/resource/SIO_000904,Visualisations that are displayed on page
+Charts,,,chart id,string,1,TRUE,,,,,http://semanticscience.org/resource/SIO_000115,"A unique value that identifies a visualisation, the associated settings and data points."
```

Comment:
I see you like to have the table name in column name? Gets quite long then, right?

Reply:
I think this is the same as having the `semantics.csv`(?)